prowler/__main__.py

```diff
@@ -280,9 +282,29 @@ def prowler():
             )
             sys.exit(1)
 
+    # Outputs
+    # TODO: this part is needed since the checks generates a Check_Report_XXX and the output uses Finding
+    # This will be refactored for the outputs generate directly the Finding
+    finding_outputs = []
+    for finding in findings:
+        finding_outputs.append(Finding.generate_output(global_provider, finding))
```

Comment:
List comprehension is more efficient and in this case it is readable enough.

```suggestion
finding_outputs = [Finding.generate_output(global_provider, finding) for finding in findings]
```

Reply:
Done